### ISSUE TITLE
Does CTK support C-MOVE?

### ISSUE THREAD

**Issue by CodingFatman** [commontk/CTK#812]
CTK supply many useful classes, but I can not find a class like DcmStorageSCP, the ctkDicomRetrieve only support C-GET, though it can send C-MOVE request, but I can not find a class deal with C-MOVE response. I try to use DcmStorageSCP to start a scp storage server, but it needs many additional work like manully deal with ctkDicomDatabase etc. is there some class to wrap the C-MOVE operation, and could be directly used?

**Comment by jcfr** [comment]
cc: @pieper @nolden

**Comment by pieper** [comment]
Right, there's no C-STORE SCP wrapper in CTK.  In [Slicer we use DCMTK's storescp executable](https://github.com/Slicer/Slicer/blob/master/Modules/Scripted/DICOMLib/DICOMProcesses.py#L127-L273) started via QProcess, to integrate it into the event loop.  Since it is a server it needs to listen on the network socket and notify the main app when data is available.

**Comment by CodingFatman** [comment]
Thank you. finally I use the class DcmStorageSCP of dcmtk to act as a store scp, then I use the signal studiesRetrieved of ctkDICOMQueryRetrieveWidget to import(use ctkDICOMIndexer) c-move received files.

  it seems the parameter of signal studiesRetrieved are not provided, the value is still empty. I use ctkDICOMTableManager of ctkDICOMQueryRetrieveWidget instead to get  currentStudiesSelection
